lib/install/elm.rb

```diff
@@ -19,4 +19,7 @@
 run "./bin/yarn add --dev elm-hot-loader elm-webpack-loader"
 run "yarn run elm package install -- --yes"
 
+puts "Updating elm source location"
+gsub_file "elm-package.json", /\"\.\"\n/, %("app/javascript/packs"\n)
```

Comment:
Yeah, that would be nice but I'm afraid that method yields the absolute path. We need a relative path here because if you happen to move the generated Rails app to a different folder, your Elm app will fail to compile. Example:

```
# elm-package.json with Webpacker::Configuration.entry_path:
{
    "source-directories": [
        "/Users/bkuhlmann/Downloads/example/app/javascript/packs"
    ]
}

# elm-package.json without Webpacker::Configuration.entry_path (using code in this PR):
{
    "source-directories": [
        "app/javascript/packs"
    ]
}

```